portal/models/i18n_utils.py

```diff
@@ -302,3 +308,177 @@ def compile_pos():
             "Saved MO file: %s",
             os.path.relpath(mo_filepath, current_app.root_path),
         )
+
+
+def upsert_to_template_file():
+    db_translatables = {}
+    db_translatables.update(get_db_strings())
+    if not db_translatables:
+        current_app.logger.warn("no DB strings extracted")
+        return
```

Comment:
this looks like more than a `warn` if it causes premature exit??

Reply:
I think at one point we didn't have DB strings to translate from TNUSA